tests/metafake.h

```diff
@@ -10,9 +10,6 @@
 #include "utilities/utils.h"
 #include "uptane_repo.h"
 
-enum BackupList {
-};
-
```

Comment:
BTW I also did some cleanup of this file in https://github.com/advancedtelematic/aktualizr/pull/1296/commits/63ce676d075635447ee3bfcde5de76d07a6d17d8.

Reply:
Ah nice! I'll remove that duplicate.

Reply:
No problem, you can also steal the commit if you want, or I can pull it out into a separate PR, since it's been hanging out in my branch for two weeks or something at this point.

Reply:
Yes I'd vote for moving it to its own PR.

Reply:
Done, see https://github.com/advancedtelematic/aktualizr/pull/1322.